packages/data/src/registry.js

```diff
@@ -143,6 +198,7 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 	let registry = {
 		registerGenericStore,
 		stores,
+		__experimentalResolveSelect,
```

Comment:
Can we move this under `select`?

Reply:
Hi @epiqueras,
You mean to have something like?
wp.data.select.__experimatalResolve( storeName )

Reply:
I meant to move the line under `select,` to match the order of the rest of the code.